ai21/models/responses/thread_response.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from typing import Optional, List, Literal
+
+from typing_extensions import TypedDict
+
+from ai21.models.ai21_base_model import AI21BaseModel
+
+
+MessageRole = Literal["assistant", "user"]
+
+
+class MessageContentText(TypedDict):
+    type: Literal["text"]
+    text: str
+
+
+class CreateMessagePayload(TypedDict):
+    role: MessageRole
+    content: MessageContentText
+
```

Comment:
I think its worth separating the non response objects to be under `models` package as other objects are